uber/templates/budget/index.html

```diff
@@ -18,50 +19,33 @@
     <h1 class="text-center">(${{ total }} total)</h1>
 </div>
 
+{% macro table_entry(label, value) -%}
+    <tr>
+        <td>{{ label }}</td>
+        <td>${{ value }}</td>
+    </tr>
+{%- endmacro %}
```

Comment:
this is another leftover part of the PR where I should have just left his page alone, but was trying to demo off some Jinja2 features.  after proving that it worked, I should have started from scratch rather than trying to do too much in this PR.  woooppppsss...... was too excited about Jinja2 to help myself.

lesson: refactor ONE THING AT A TIME. RESIST!